activitysim/defaults/models/mode.py

```diff
@@ -0,0 +1,124 @@
+import os
+import copy
+import yaml
+import pandas as pd
+import urbansim.sim.simulation as sim
+from activitysim import activitysim as asim
+from activitysim import skim as askim
+
+"""
+Mode choice is run for all tours to determine the transportation mode that
```

Comment:
Just a note that mode choice happens for tours and trips, so eventually you will either be generic in this file to accept either or create something very similar (in which case you may want to rename this file).  This isn't urgent...just something to think about.